common/src/test/java/tachyon/util/io/FileUtilsTest.java

```diff
@@ -70,4 +70,32 @@ public void changeLocalDirPermissionTests() throws IOException {
     FileUtils.changeLocalFilePermission(mTestFolder.getRoot().getAbsolutePath(), "744");
     Assert.assertTrue(tempFile.delete());
   }
+
+  @Test
+  public void moveFileTest() throws IOException {
+    File fromFile = mTestFolder.newFile("from.txt");
+    File toFile = mTestFolder.newFile("to.txt");
+    // Move a file and verify
+    FileUtils.move(fromFile, toFile);
+    Assert.assertFalse(fromFile.exists());
+    Assert.assertTrue(toFile.exists());
+  }
+
+  @Test
+  public void deleteFileTest() throws IOException {
+    File tempFile = mTestFolder.newFile("toDelete.txt");
+    File tempFolder = mTestFolder.newFolder("temp-dir");
+    // Delete a file and a directory
+    Assert.assertTrue(FileUtils.delete(tempFile));
+    Assert.assertTrue(FileUtils.delete(tempFolder));
+    Assert.assertFalse(tempFile.exists());
+    Assert.assertFalse(tempFolder.exists());
+  }
+
+  @Test
+  public void deleteNoExistentFileTest() throws IOException {
```

Comment:
I think ghostFile is an empty File instance which cannot tell the difference of file or dir until createNewFile() or mkdir() is called?